src/ccl_musigma.c

```diff
@@ -11,6 +11,51 @@
 #include "ccl.h"
 
 
+/* --------- ROUTINE: ccl_mu_MG ---------
+INPUT: cosmology object, scale factor, wavenumber for scale
+TASK: Compute mu(a,k) where mu is one of the the parameterizating functions
+of modifications to GR in the quasistatic approximation.
+*/
+
+double ccl_mu_MG(ccl_cosmology * cosmo, double a, double k, int *status)
+{
+    double s1_k, s2_k, hnorm;
+	// This function can be extended to include other
+	// redshift and scale z-dependences for mu in the future
+    if (k==0.0) {
+        s1_k = cosmo->params.c1_mg;
+    }
+    else {
+      hnorm = ccl_h_over_h0(cosmo, a, status);
+	    s2_k = (cosmo->params.lambda_mg*(hnorm*cosmo->params.H0)/k/(ccl_constants.CLIGHT/1000));
+	    s1_k = (1.0+cosmo->params.c1_mg*s2_k*s2_k)/(1.0+s2_k*s2_k);
+	}
+	return cosmo->params.mu_0 * ccl_omega_x(cosmo, a, ccl_species_l_label, status)/cosmo->params.Omega_l*s1_k;
```

Comment:
You are right but I would like to suggest to keep it for now just keeps the code balanced and in case we need it for something else before we put the new API. I hope you do not have a strong feeling about this one and will let me keep this.

Reply:
No, that's fine!